packages/drivers/playwright/examples/example.test.ts

```diff
@@ -9,8 +10,9 @@ describe("Example Test Suite", () => {
 
   beforeAll(async () => {
     const promptHandler: PromptHandler = new PromptHandler();
-
-    frameworkDriver = new PlaywrightFrameworkDriver();
+    const driverUtils: WebTestingFrameworkDriverUtils =
+      new WebTestingFrameworkDriverUtils();
+    frameworkDriver = new PlaywrightFrameworkDriver(driverUtils);
```

Comment:
The user should not pass this to the constructor as a param, it should be created on the constructor